scripts/utils/create-jest-config.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+module.exports = (resolve, rootDir) => {
+  const config = {
+    automock: false,
```

Comment:
I really like that this turns off automocking by default. Do you think Jest itself should / will soon switch that to the default behavior for everyone? It might confuse people when, if they start with `create-react-app` and use Jest, the behavior is a lot different than if they just start with Jest directly.